src/txkube/test/test_authentication.py

```diff
@@ -358,7 +358,9 @@ def test_creatorForNetLoc_interface(self, host_known, host_used, port_known, por
                 netloc: cert,
             },
         )
-        creator = policy.creatorForNetloc(host_used, port_used)
+        creator = policy.creatorForNetloc(
+            host_used.encode("ascii"),
+            port_used)
```

Comment:
Please uncuddle the closing `)` and add a trailing `,` after `port_used`.

Reply:
I don't understand what you mean by uncuddle here.  Can you clarify?

Reply:
OK, I did a web search for cuddling indentation, and I think I understand, so I updated the patch.